nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/sqs/GetSQS.java

```diff
@@ -64,6 +66,7 @@
     @WritesAttribute(attribute = "sqs.message.id", description = "The unique identifier of the SQS message"),
     @WritesAttribute(attribute = "sqs.receipt.handle", description = "The SQS Receipt Handle that is to be used to delete the message from the queue")
 })
+@DefaultSchedule(strategy = SchedulingStrategy.TIMER_DRIVEN, period = "1 min")
```

Comment:
Similar to `GetAzureEventHub`, it seems like this Processor needs to be run more often. Recommend removing the annotation from this set of changes and revisiting.